internal/databases/postgres/tar_ball_composer.go

```diff
@@ -57,6 +61,25 @@ func NewTarBallComposerMaker(composerType TarBallComposerType, conn *pgx.Conn,
 			return nil, err
 		}
 		return NewRatingTarBallComposerMaker(relFileStats, filePackOptions)
+	case CopyComposer:
+		previousBackupName, err := internal.GetLatestBackupName(folder)
+		if err != nil {
+			return NewRegularTarBallComposerMaker(filePackOptions), nil
```

Comment:
Add some  tracelog.InfoLogger output to indicate that regular composer is going to be used instead of the copy.

Reply:
I see no change here